sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -320,14 +322,6 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
     }
   }
 
-  private fun releaseEgl() {
```

Comment:
noting that this function is not safe in general - releasing EGL _without_ recreating native renderer may result in native renderer pointing at outdated resources